Directory.Packages.props

```diff
@@ -1,32 +1,30 @@
 <Project>
-  <!-- <Import Project="eng\Versions.props" /> -->
 
   <PropertyGroup>
-    <!-- In order tests against the same version of NuGet as the SDK. We have to set this to match. -->
-    <NuGetVersion>6.4.0</NuGetVersion>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
```

Comment:
❔ Is this file always included, or only included when `ManagePackageVersionsCentrally` is set? Just want to make sure it's still applied.

Reply:
➡️ It's always imported.
https://github.com/NuGet/NuGet.Client/blob/476dca9762230d3212162b4be3f8d77151678711/src/NuGet.Core/NuGet.Build.Tasks/NuGet.props#L35-L39

Reply:
Yes. `ManagePackageVersionsCentrally` is usually set directly in Directory.Packages.props and components that don't want to auto-import that file are expected to set `ImportDirectoryPackagesProps=false`.